src/components/editor/property/InputLiteral.jsx

```diff
@@ -285,8 +275,8 @@ const mapDispatchToProps = dispatch => ({
   handleMyItemsChange(userInput) {
     dispatch(setItems(userInput))
   },
-  handleRemoveItem(id) {
-    dispatch(removeItem(id))
+  handleRemoveItem(reduxPath, itemId) {
+    dispatch(removeItem(reduxPath, itemId))
```

Comment:
my spotting redux and sinopia-redux knowledge says. so we need the redux path so we can scope the item id and/or ensure we are hooking up to the correct place in the nested resource template madness.  true?

Reply:
yes, the reduxPath and id in combination is the address of the data element to remove